azerothcore/azerothcore-wotlk
# fix(Core/Script): Molten Core: Lucifron

Co-Author: @sanctum32 (Code)
Co-Author: @T1ti (Research)
Co-Author: @Hacki95 (Coordination)



## Changes Proposed:
- Correct all timers to the researched values.
- Randomize the timers.

## Issues Addressed:

- Closes #5456

## SOURCE:

- Research in #5456

## Tests Performed:

- Fix applied
- Fought agains Lucifron


## How to Test the Changes:


1. Apply the Fix
2. .tele moltencore
3. Fight Lucifron
4. Check the timers and spells.

## Known Issues and TODO List:


- /


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
